app/javascript/components/subject/AssignedUser.js

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Button, Modal, Form } from 'react-bootstrap';
+import _ from 'lodash';
+import axios from 'axios';
+
+import InfoTooltip from '../util/InfoTooltip';
+import reportError from '../util/ReportError';
+
+class AssignedUser extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showAssignedUserModal: false,
+      assigned_user: props.patient.assigned_user ? props.patient.assigned_user : '',
```

Comment:
Small nitpick, but I've seen it elsewhere in the code. You can do `props.patient.assigned_user || ''` here.

Reply:
done